perfkitbenchmarker/providers/openstack/os_virtual_machine.py

```diff
@@ -54,21 +55,47 @@ def __init__(self, vm_spec):
                                                FLAGS.run_uri)
         self.client = os_utils.NovaClient()
         self.public_network = os_network.OpenStackPublicNetwork(
-            FLAGS.openstack_public_network
+            FLAGS.openstack_floating_ip_pool
```

Comment:
Does it make sense to instantiate `self.public_network` if openstack_floating_ip_pool is None?